### PR TITLE
Add mutual icon on left side option

### PR DESCRIPTION
**Pull request by Wolvan**
Some people may not like the positioning of the mutual icon to be on the
right, personally with the reblog icon it feels a bit cluttered to me,
so I added this option to put it on the left side for people that
prefer that.

### CODE REVIEW

File: Extensions/mutualchecker.js [hunk 1]
```diff
@@ -17,6 +17,12 @@ XKit.extensions.mutualchecker = new Object({
 			default: "",
 			value: "",
 			type: "blog"
+		},
+		"put_in_front": {
+			text: "Place mutual icon in front",
+			default: false,
+			value: false,
+			type: "checkbox"
```
Comment (AprilSylph) [inline Extensions/mutualchecker.js]: type is unnecessary here

File: Extensions/mutualchecker.js [hunk 2]
```diff
@@ -95,9 +101,10 @@ XKit.extensions.mutualchecker = new Object({
 
 	add_label: function(name_div, user) {
 		if ($(name_div).hasClass("post_info_submissions")) {
-			name_div.html('<span class="mutuals">' + user + '</span>' + name_div.text().trim().substring(user.length));
+			name_div.html('<span class="mutuals' + ( XKit.extensions.mutualchecker.preferences.put_in_front.value ? " mutuals-front" : "") + '">' + user + '</span>' + name_div.text().trim().substring(user.length));
 		} else {
 			name_div.addClass("mutuals").attr("title", user + " follows you");
+			if (XKit.extensions.mutualchecker.preferences.put_in_front.value) name_div.addClass("mutuals-front");
```
Comment (AprilSylph) [inline Extensions/mutualchecker.js]: this `if` should be in braces

File: Extensions/mutualchecker.css [hunk 3]
```diff
@@ -1,8 +1,16 @@
-.mutuals::after{
+.mutuals::after, .mutuals::before{
```
Comment (AprilSylph) [inline Extensions/mutualchecker.css]: shouldn't this just be `.mutuals` instead? any pseudo-elements should use their parent's css afaik

Comment (Wolvan) [reply Extensions/mutualchecker.css]: If I were to use just mutuals, wouldn't it affect the text itself as well? We only want the before and after styled, iirc

Comment (AprilSylph) [reply Extensions/mutualchecker.css]: oh right. forgot i changed it to not be its own element, whoops

File: Extensions/mutualchecker.js [hunk 4]
```diff
@@ -17,6 +17,11 @@ XKit.extensions.mutualchecker = new Object({
 			default: "",
 			value: "",
 			type: "blog"
+		},
+		"put_in_front": {
+			text: "Place mutual icon in front",
```
Comment (AprilSylph) [inline Extensions/mutualchecker.js]: minor nitpick but I feel like this option could be more descriptive - "Place mutual icon before, not after usernames" or something similar

**Pull request merged**